src/main/java/frc/team2412/robot/util/LauncherDataLoader.java

```diff
@@ -0,0 +1,116 @@
+package frc.team2412.robot.util;
+
+import edu.wpi.first.math.interpolation.InterpolatingTreeMap;
+import edu.wpi.first.math.interpolation.InverseInterpolator;
+import edu.wpi.first.wpilibj.Filesystem;
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.function.Consumer;
+
+public class LauncherDataLoader {
+
+	public static InterpolatingTreeMap<Double, LauncherDataPoint> fromCSV(Path path) {
+		InterpolatingTreeMap<Double, LauncherDataPoint> map =
+				new InterpolatingTreeMap<>(InverseInterpolator.forDouble(), LauncherDataPoint.INTERPOLATOR);
+
+		try {
+			BufferedReader reader = Files.newBufferedReader(path);
+			BufferedWriter debugWriter =
+					Files.newBufferedWriter(
+							FileSystems.getDefault()
+									.getPath(Filesystem.getOperatingDirectory().getPath(), "logs/CSVInterpreter.log"),
+							StandardOpenOption.CREATE,
+							StandardOpenOption.WRITE);
```

Comment:
I'd recommend using a try-with-resources (`try (BufferedReader reader = ...; BufferedWriter debugWriter = ...) { ... } catch { ... }`) to ensure these get closed properly, even in the case of an exception. (And also remove the manual `close()` call towards the end- It's documented that repeated `close()` calls won't do anything bad, but it's still better to avoid it)